content/manuals/engine/release-notes/27.md

```diff
@@ -23,6 +23,78 @@ For more information about:
+- containerd image store: Fix Docker daemon failing to fully start with a "context deadline exceeded error" with containerd snapshotter and many builds/images. [moby/moby#48954](https://github.com/moby/moby/pull/48954)
+- containerd image-store: Fix partially pulled images not being garbage-collected. [moby#48910](https://github.com/moby/moby/pull/48910), [moby/moby#48957](https://github.com/moby/moby/pull/48957)
+- containerd image store: Fix `docker image inspect` outputting duplicate references in `RepoDigests`. [moby/moby#48785](https://github.com/moby/moby/pull/48785)
+- containerd image store: Fix not being able to connect to some insecure registries in cases where the HTTPS request failed due to a non-TLS related error. [moby/moby#48758](https://github.com/moby/moby/pull/48758)
+- containerd image store: Remove a confusing warning log when tagging a non-dangling image. [moby/moby#49010](https://github.com/moby/moby/pull/49010)
+- dockerd-rootless-setuptool.sh: let --force ignore smoke test errors [moby/moby#48695](https://github.com/moby/moby/pull/48695)
+- Disable IPv6 Duplicate Address Detection (DAD) for addresses assigned to the bridges belonging to bridge networks. [moby/moby#48684](https://github.com/moby/moby/pull/48684)
+- Remove BuildKit init timeout. [moby/moby#48963](https://github.com/moby/moby/pull/48963)
+- Ignore "dataset does not exist" error when removing dataset on ZFS. [moby/moby#48968](https://github.com/moby/moby/pull/48968)
+- Client: Prevent idle connections leaking FDs. [moby/moby#48764](https://github.com/moby/moby/pull/48764)
+- Fix anonymous volumes being created through the `--mount` option not being marked as anonymous. [moby/moby#48755](https://github.com/moby/moby/pull/48755)
+- After a daemon restart with live-restore, ensure an iptables jump to the `DOCKER-USER` chain is placed before other rules. [moby/moby#48714](https://github.com/moby/moby/pull/48714)
+- Fix a possible memory leak caused by OTel meters. [moby/moby#48693](https://github.com/moby/moby/pull/48693)
+- Create distinct build history db for each store. [moby/moby#48688](https://github.com/moby/moby/pull/48688)
+- Fix an issue that caused excessive memory usage when DNS resolution was made in a tight loop. [moby/moby#48840](https://github.com/moby/moby/pull/48840)
+- containerd image store: Do not underline names in `docker image ls --tree`. [docker/cli#5519](https://github.com/docker/cli/pull/5519)
+- containerd image store: Change name of `USED` column in `docker image ls --tree` to `IN USE`. [docker/cli#5518](https://github.com/docker/cli/pull/5518)
+- Fix a bug preventing image pulls from being cancelled during `docker run`. [docker/cli#5654](https://github.com/docker/cli/pull/5654)
+- Port some completions from the bash completion to the new cobra based completion. [docker/cli#5618](https://github.com/docker/cli/pull/5618)
```

Comment:
Does this need to be in the release notes?

```suggestion
- Port some completions from the bash completion to the new Cobra-based completion. [docker/cli#5618](https://github.com/docker/cli/pull/5618)
```

Reply:
I see I copied this as-is from the original PR, but perhaps  "shell completion" (instead of bash)? Or we should leave out the "ported", which may not be super relevant to the user, as it's effectively just adding more completions to the Cobra-based completion.

Reply:
In that case, I think we can just drop this line completely